src/transformStateWithClones.js

```diff
@@ -6,8 +6,46 @@
  *
  * @return {Object[]}
  */
+
 function transformStateWithClones(state, actions) {
-  // write code here
+  const allStates = [];
+  const realState = Object.assign({}, state);
```

Comment:
As per the checklist item [NAMING], the name `realState` should be more descriptive to reflect that it is a copy of the state. A better name would be `stateCopy` or similar.